JSDemos/Demos/TreeList/ColumnChooser/description.md

```diff
@@ -1 +1,13 @@
-The Column Chooser is&nbsp;an&nbsp;element that allows a&nbsp;user to&nbsp;alter the set of&nbsp;visible columns at&nbsp;runtime. To&nbsp;open the column chooser, click the _&laquo;Column Chooser&raquo;_ button at&nbsp;the top of&nbsp;TreeList. Now, you can drag and drop column headers to&nbsp;and from the TreeList, thus showing or&nbsp;hiding columns. Use the column chooser&rsquo;s search box to&nbsp;quickly find the required column.
\ No newline at end of file
+The column chooser allows users to hide and unhide columns at runtime. To enable it, assign **true** to the **columnChooser**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#enabled) property. 
+
+To open the column chooser, click the button in the toolbar above the TreeList component. The opened popup panel lists the headers of hidden columns. In this demo, you can specify the way users interact with this panel to hide/unhide columns. For this, use the drop-down list below the TreeList component to set the [mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#mode) property to one of the following values:
+
+* **"dragAndDrop"**
+Users have to drag and drop column headers to and from the column chooser panel.
+
+* **"select"**
+Users have to choose checkboxes to select visible columns.
+
+If the [allowSearch](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#allowSearch) is set to **true**, you can search through the column chooser values. In this demo, use the checkbox below the TreeList to toggle this feature.
+
+You can also hide certain columns programmatically. For this, set **columns[]**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#visible) to **false**. If a column should never be hidden, assign **false** to **columns[]**.[allowHiding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHiding). 
```

Comment:
The word 'certain' is banned by Ray. I can't find the exact place, so [here](https://teams.microsoft.com/l/message/19:78e9fcaa3c734fc3a95cdfe1bbb57f41@thread.skype/1586327699676?tenantId=e4d60396-9352-4ae8-b84c-e69244584fa4&groupId=5afdc900-1758-4beb-8368-d4d641dc99ea&parentMessageId=1586327699676&teamName=Let%20us%20speak%20English%20and%20discuss%20it%20here!&channelName=General&createdTime=1586327699676) is the first post with replacement I found

```suggestion
You can also hide specific columns programmatically. For this, set **columns[]**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#visible) to **false**. If a column should never be hidden, assign **false** to **columns[]**.[allowHiding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHiding). 
```